packages/cf.js/src/types/simple-types.ts

```diff
@@ -2,3 +2,6 @@ export type ABIEncoding = string;
 export type AppInstanceID = string;
 export type Address = string;
 export type Bytes32 = string;
+
+export type AppState = object;
+export type AppAction = object;
```

Comment:
Technically these could also be simple `numbers` or `bytes` which wouldn't be translated into Typescript as `object`s. Better to make these `any`